site/src/pages/OrganizationSettingsPage/OrganizationSettingsPage.tsx

```diff
@@ -0,0 +1,185 @@
+
+  const { currentOrganizationId, organizations } = useOrganizationSettings();
+
+  const org = organizations.find((org) => org.id === currentOrganizationId)!;
+
+  const error =
+    addOrganizationMutation.error ?? deleteOrganizationMutation.error;
+
+  const form = useFormik<UpdateOrganizationRequest>({
+    initialValues: {
+      name: org.name,
+      display_name: org.display_name,
+      description: org.description,
+      icon: org.icon,
+    },
+    validationSchema: getValidationSchema(),
+    onSubmit: (values) =>
+      updateOrganizationMutation.mutateAsync({ orgId: org.id, req: values }),
+    enableReinitialize: true,
+  });
+  const getFieldHelpers = getFormHelpers(form, error);
+
+  const [newOrgName, setNewOrgName] = useState("");
+
+  return (
+    <Margins verticalMargin={18}>
+      {Boolean(error) && <ErrorAlert error={error} />}
+
+      <PageHeader css={{ paddingTop: 0 }}>
+        <PageHeaderTitle>Organization settings</PageHeaderTitle>
+      </PageHeader>
+
+      <HorizontalForm
+        onSubmit={form.handleSubmit}
+        aria-label="Template settings form"
+      >
+        <FormSection
+          title="General info"
+          description="Change the name or description of the organization."
+        >
+          <FormFields>
+            <TextField
+              {...getFieldHelpers("name")}
+              disabled={form.isSubmitting}
+              onChange={onChangeTrimmed(form)}
+              autoFocus
+              fullWidth
+              label="Name"
+            />
+            <TextField
+              {...getFieldHelpers("display_name")}
+              disabled={form.isSubmitting}
+              fullWidth
+              label="Display name"
+            />
+            <TextField
+              {...getFieldHelpers("description")}
+              multiline
+              disabled={form.isSubmitting}
+              fullWidth
+              label="Description"
+              rows={2}
+            />
+            <IconField
+              {...getFieldHelpers("icon")}
+              disabled={form.isSubmitting}
+              onChange={onChangeTrimmed(form)}
+              fullWidth
+              onPickEmoji={(value) => form.setFieldValue("icon", value)}
+            />
+          </FormFields>
+        </FormSection>
+        <FormFooter isLoading={form.isSubmitting} />
+      </HorizontalForm>
+
+      {!org.is_default && (
+        <Button
+          css={styles.dangerButton}
+          variant="contained"
+          onClick={() =>
+            deleteOrganizationMutation.mutate(currentOrganizationId)
+          }
+        >
+          Delete this organization
+        </Button>
+      )}
+
+      <Stack css={{ marginTop: 128 }}>
+        <TextField
+          label="New organization name"
+          onChange={(event) => setNewOrgName(event.target.value)}
+        />
+        <Button
+          onClick={() => addOrganizationMutation.mutate({ name: newOrgName })}
+        >
+          Create new team
```

Comment:
I feel the new team form should be outside of `!org.is_default` right? I'm trying to create a new team but since the first org is always the default it would never show.

Reply:
it isn't in that condition. it sounds like you don't have the experiment flag set.

Reply:
I'm running it with: `CODER_EXPERIMENTS=*,multi-organization ./scripts/develop.sh` is that correct?

Reply:
I had it but since the form was below the fold I didn't see it 🤦.